src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -31,7 +31,8 @@ static DatadogLogging()
             try
             {
                 // We use environment variables and not the tracer settings to avoid a startup race condition between the logger and the tracer.
```

Comment:
Does this mean debug logging is only enabled with environment variables, not through other configuration sources?

Reply:
Yes

Reply:
I'd like to solve this, but it's an existing problem, can we do it in a follow up?

Reply:
yup, looks like it's been there for a while.

Reply:
Actually, not that long. We should make it clear in the documentation and to the support team.